### PR TITLE
ERC20 PostgreSQL integration

### PR DESCRIPTION
**Pull request by NickSneo**
Added postgreSQL integration with ERC20 tool
Added reading contract address from Postgres DB feature

### CODE REVIEW

**Comment by dparmar** [comment]
Considering the simplicity of the requirement, JPA is an overkill. Checkout https://www.sql2o.org/

File: web3j/pom.xml [hunk 1]
```diff
@@ -36,6 +36,11 @@
       <artifactId>spring-boot-starter-validation</artifactId>
     </dependency>
 
+    <dependency>
```
Comment (dparmar) [inline web3j/pom.xml:39]: Considering the simplicity of the requirement, JPA is an overkill. Checkout https://www.sql2o.org/

**Review by dparmar — CHANGES_REQUESTED** [review]
Make it simpler, use sql2o.

File: web3j/pom.xml (hunk 1 above)
Comment (NickSneo) [reply web3j/pom.xml:39]: I was checking out sql2o, actually, ramki has used hstore data type, in JPA it was simple to implement it in contract model. But for sql2o I will need to figure it out.

**Pull request closed without merging**